src/librustdoc/html/highlight.rs

```diff
@@ -687,15 +711,30 @@ fn string<T: Display>(
     out: &mut Buffer,
     text: T,
     klass: Option<Class>,
-    context_info: &Option<ContextInfo<'_, '_, '_>>,
+    href_context: &Option<HrefContext<'_, '_, '_>>,
 ) {
+    if let Some(closing_tag) = string_without_closing_tag(out, text, klass, href_context) {
+        out.write_str(closing_tag);
+    }
+}
+
+fn string_without_closing_tag<T: Display>(
```

Comment:
Good point, I'll add a doc comment and rename it.

Reply:
I sopke too fast, for once I think it's a good name because it's based on the `string` function just above. I can rename both though if you have an idea for the name to use.